NeuralAmpModeler/resources/main.rc

```diff
@@ -239,3 +239,6 @@ HELP_FN SVG HELP_FN
 /////////////////////////////////////////////////////////////////////////////
 #endif    // not APSTUDIO_INVOKED
 
+DOWN_ARROW_FN SVG DOWN_ARROW_FN  
```

Comment:
Comment about the arrow graphics: can the insides be white? Otherwise they're hard to see.

Reply:
yes, please make it more usable :)